### PR TITLE
EXP-747 extrato - criar validação para exibição das informaçoes de liquidação externa

### PR DESCRIPTION
**Pull request by LucasRichter**
## Contexto
Esse PR visa possibilitar que o usuário seja informado no extrato caso sua company possua contratos de recebiveis externos.

## Checklist
- [ ] Exibir a informação no card “A receber” 
- [ ] Ao clicar em "Saiba mais", exibir o modal
- [ ]  Lembrar de corrigir a variável `hasExternalContract`

### Layout:
image

## Como testar?
- Verificar se ao clicar em "Saiba mais", abra a modal

### CODE REVIEW

**Comment by GitCop** [comment]
There were the following issues with your Pull Request

 * Commit: d064961a5d6521ea690f5082fc73904202580587
  * lways use the standard `scope: action description`, sempre em inglês  
    - The **scope** shouldn't have spaces. If the word is composite, use dashes.  
    - The **action** should be **add**, **remove**, **refactor**, **move**,  **bump**, **update** or **release**  
    - The **description** should contain a quick description of the change

The complete commit style guide is available at https://github.com/pagarme/git-style-guide

---
This message was auto-generated by https://gitcop.com

**Comment by GitCop** [comment]
There were the following issues with your Pull Request

 * Commit: d064961a5d6521ea690f5082fc73904202580587
  * lways use the standard `scope: action description`, sempre em inglês  
    - The **scope** shouldn't have spaces. If the word is composite, use dashes.  
    - The **action** should be **add**, **remove**, **refactor**, **move**,  **bump**, **update** or **release**  
    - The **description** should contain a quick description of the change

The complete commit style guide is available at https://github.com/pagarme/git-style-guide

---
This message was auto-generated by https://gitcop.com

**Comment by GitCop** [comment]
There were the following issues with your Pull Request

 * Commit: d064961a5d6521ea690f5082fc73904202580587
  * lways use the standard `scope: action description`, sempre em inglês  
    - The **scope** shouldn't have spaces. If the word is composite, use dashes.  
    - The **action** should be **add**, **remove**, **refactor**, **move**,  **bump**, **update** or **release**  
    - The **description** should contain a quick description of the change

The complete commit style guide is available at https://github.com/pagarme/git-style-guide

---
This message was auto-generated by https://gitcop.com

**Comment by GitCop** [comment]
There were the following issues with your Pull Request

 * Commit: d064961a5d6521ea690f5082fc73904202580587
  * lways use the standard `scope: action description`, sempre em inglês  
    - The **scope** shouldn't have spaces. If the word is composite, use dashes.  
    - The **action** should be **add**, **remove**, **refactor**, **move**,  **bump**, **update** or **release**  
    - The **description** should contain a quick description of the change

The complete commit style guide is available at https://github.com/pagarme/git-style-guide

---
This message was auto-generated by https://gitcop.com

File: packages/pilot/src/containers/Balance/index.js [hunk 1]
```diff
@@ -588,11 +608,16 @@ class Balance extends Component {
                   //   || anticipationError
                   //   || available < MINIMUM_API_VALUE
                   // }
-                  detail={!isCompanyPaymentLink && (
-                    <span>
-                      {t('pages.balance.anticipation_call')}
-                    </span>
-                  )}
+                  detail={
```
Comment (Hrfreire) [inline packages/pilot/src/containers/Balance/index.js:611]: Acho que aqui, talvez fosse melhor criar uma função pra renderizar toda essa parte de detail.

Acho que esse ternário concatenado com &&, fica um pouco confuso de ler.

File: packages/pilot/src/pages/Balance/Balance.js [hunk 2]
```diff
@@ -261,7 +261,7 @@ const areEqualPages = (query, search) => {
 
 const defaultPageNumber = 15
 
-class Balance extends Component {
+export class Balance extends Component {
```
Comment (Hrfreire) [inline packages/pilot/src/pages/Balance/Balance.js:264]: Não entendi esse export aqui. A page já não é exportada com default no final do arquivo?

File: packages/pilot/src/containers/Balance/index.js [hunk 3]
```diff
@@ -149,6 +150,7 @@ class Balance extends Component {
         end,
         start,
       },
+      modallExternalContractOpen: false,
```
Comment (diegodsgarcia) [inline packages/pilot/src/containers/Balance/index.js:153]: typo `modal` com dois `ll`

File: packages/pilot/src/containers/Balance/index.js [hunk 4]
```diff
@@ -501,6 +520,7 @@ class Balance extends Component {
 
     const isCompanyPaymentLink = isPaymentLink(company)
     const balanceGridCol = isCompanyPaymentLink ? 6 : 4
+    const hasExternalContract = true // TODO: alterar para company.capabilities?.<A_DEFINIR>
```
Comment (diegodsgarcia) [inline packages/pilot/src/containers/Balance/index.js:523]: Aqui já não vale criar uma function que retorna um boolean, já que iremos agora pegar essa "capability" de uma rota?

File: packages/pilot/src/locales/pt/translations.json [hunk 5]
```diff
@@ -976,6 +976,8 @@
           "excel": "Excel"
         }
       },
+      "external_contract_message": "Você tem valores comprometidos em outro domicílio bancário. ",
```
Comment (diegodsgarcia) [inline packages/pilot/src/locales/pt/translations.json:979]: Tem um espaço branco no final da string. É necessário?

File: packages/pilot/src/containers/Balance/style.css [hunk 6]
```diff
@@ -19,3 +20,24 @@
...
+.externalContractButton {
+  appearance: none;
+  border: none;
+  background: transparent;
+  font-size: 14px;
+  line-height: 17px;
+  padding: 0;
+  color: var(--color-picklerick-green-100);
+  font-weight: 600;
+  transition: opacity 0.124s ease-in-out;
```
Comment (diegodsgarcia) [inline packages/pilot/src/containers/Balance/style.css:37]: Será que não da pra arredondar pra 0.1s? 0.124s é bem especifico hahaha

File: packages/pilot/src/containers/Balance/style.css [hunk 7]
```diff
@@ -19,3 +20,24 @@
   text-align: center;
   width: 100%;
 }
+
+.externalContractContainer {
+  margin-top: 9px;
```
Comment (diegodsgarcia) [inline packages/pilot/src/containers/Balance/style.css:25]: Aqui vale colocar a variavel `var(--spacing-tiny)`

File: packages/pilot/src/containers/Balance/style.css [hunk 8]
```diff
@@ -19,3 +20,24 @@
...
+}
+
+.externalContractButton {
+  appearance: none;
+  border: none;
+  background: transparent;
+  font-size: 14px;
```
Comment (diegodsgarcia) [inline packages/pilot/src/containers/Balance/style.css:32]: Acho que esse `font-size` não é necessário por default ser 14px mesmo

File: packages/pilot/src/pages/Balance/Balance.js (hunk 2 above)
Comment (LucasRichter) [reply packages/pilot/src/pages/Balance/Balance.js:264]: Eu exportei pra poder testar sem os middlewares.

File: packages/pilot/src/containers/Balance/index.js (hunk 4 above)
Comment (LucasRichter) [reply packages/pilot/src/containers/Balance/index.js:523]: Acho melhor deixar pra fazer quando a api estiver pronta. Pra não perder o contexto.

**Comment by GitCop** [comment]
There were the following issues with your Pull Request

 * Commit: 766cecabc49899ab122486bfd0e2b6bb05d93ec3
  * lways use the standard `scope: action description`, sempre em inglês  
    - The **scope** shouldn't have spaces. If the word is composite, use dashes.  
    - The **action** should be **add**, **remove**, **refactor**, **move**,  **bump**, **update** or **release**  
    - The **description** should contain a quick description of the change

The complete commit style guide is available at https://github.com/pagarme/git-style-guide

---
This message was auto-generated by https://gitcop.com

**Comment by GitCop** [comment]
There were the following issues with your Pull Request

 * Commit: 766cecabc49899ab122486bfd0e2b6bb05d93ec3
  * lways use the standard `scope: action description`, sempre em inglês  
    - The **scope** shouldn't have spaces. If the word is composite, use dashes.  
    - The **action** should be **add**, **remove**, **refactor**, **move**,  **bump**, **update** or **release**  
    - The **description** should contain a quick description of the change

The complete commit style guide is available at https://github.com/pagarme/git-style-guide

---
This message was auto-generated by https://gitcop.com

**Comment by GitCop** [comment]
There were the following issues with your Pull Request

 * Commit: 766cecabc49899ab122486bfd0e2b6bb05d93ec3
  * lways use the standard `scope: action description`, sempre em inglês  
    - The **scope** shouldn't have spaces. If the word is composite, use dashes.  
    - The **action** should be **add**, **remove**, **refactor**, **move**,  **bump**, **update** or **release**  
    - The **description** should contain a quick description of the change

The complete commit style guide is available at https://github.com/pagarme/git-style-guide

---
This message was auto-generated by https://gitcop.com

**Comment by GitCop** [comment]
There were the following issues with your Pull Request

 * Commit: 766cecabc49899ab122486bfd0e2b6bb05d93ec3
  * lways use the standard `scope: action description`, sempre em inglês  
    - The **scope** shouldn't have spaces. If the word is composite, use dashes.  
    - The **action** should be **add**, **remove**, **refactor**, **move**,  **bump**, **update** or **release**  
    - The **description** should contain a quick description of the change

The complete commit style guide is available at https://github.com/pagarme/git-style-guide

---
This message was auto-generated by https://gitcop.com

**Review by Hrfreire — APPROVED** [review]

**Review by diegodsgarcia — APPROVED** [review]

**Pull request merged**